<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>21</y>
      <width>941</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bnt_get_new_array">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить медианный массив</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>691</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_random">
       <property name="text">
        <string>Заполнить случайными элементами, их количество: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_count_of_elements"/>
     </item>
     <item>
      <widget class="QPushButton" name="bnt_random">
       <property name="text">
        <string>Заполнить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
